<?xml version="1.0"?>

<launch>

  <node name="dawn_ik_solver" pkg="dawn_ik" type="solver_node" output="screen">
    <remap from="joint/command" to="/ufactory/lite6_traj_controller/command"/>
    <remap from="joint/state" to="/ufactory/lite6_traj_controller/state"/>
  </node>

</launch>
